### PR TITLE
[typo] Fix typo error

### PR DESCRIPTION
**Pull request by DonghakPark**
Fix the typo error through spell checker
- Application/
- api/ccapi/
- nntrainer/

Signed-off-by: DonghakPark <donghak.park@samsung.com>

### CODE REVIEW

File: nntrainer/compiler/tflite_opnode.cpp [hunk 1]
```diff
@@ -65,8 +65,15 @@ void TfOpNode::finalize() {
...
-        << "result size must match with given variable size";
+      v.resize(result.size());
+      /// basically, result.size() == v.size() except InputLayer because a
+      /// Transpose operator is added for converting nchw to nhwc
+      /// @todo comment out below codes. TfOpNode needs to have LayerNode
+      /// pointer
+      // NNTR_THROW_IF(dynamic_cast<InputLayer>(layer_ptr->getLayer()) ==
+      // nullptr && result.size() != v.size(), std::invalid_argument)
+      //   << "result size must match with given variable size";
+
```
Comment (myungjoo) [inline nntrainer/compiler/tflite_opnode.cpp:76]: This is not a typo error and requires separated discussion or review.

Comment (DonghakPark) [reply nntrainer/compiler/tflite_opnode.cpp:76]: It's my mistake.
I will fix right away

**Review by lhs8928 — APPROVED** [review]
LGTM

**Review by SeoHyungjun — APPROVED** [review]
LGTM!

**Review by myungjoo — APPROVED** [review]

**Pull request merged**